src/css/SocialsIcon.module.css

```diff
@@ -3,17 +3,24 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  width: 129px;
+  width: max-content;
+}
+
+.iconLink {
+  margin-right: 0.75rem;
 }
 
 .iconLink:hover {
-  opacity: 0.95;
+  opacity: 0.9;
```

Comment:
Good call on lowering the opacity to increase contrast. I'd say this still isn't enough, though :sweat_smile:

![image](https://user-images.githubusercontent.com/32112321/181148732-73175471-b51a-46ff-a4f2-f1b61ebcf34b.png)

Maybe 0.85 or 0.80?

Reply:
Ah yeah, I played around with the opacity for this and I think 0.85 or lower is actually too light to be distinguishable from the background 😅 I think it would be best if we could apply a different colour on hovering on this element, a colour that stands out from the background and still match Surge's branding guidelines. I'd say we will ask Design team about this issue. How do ya think?

Reply:
Sounds like a plan!